python/doc/examples/probabilistic_modeling/distributions/plot_point_conditional_distribution.py

```diff
@@ -0,0 +1,271 @@
+"""
+Create a Point Conditional Distribution
+=======================================
+"""
+
+# %%
+# In this example, we create distributions conditionned by a particular value of their marginals.
+#
+# We consider the following cases:
+#
+# - Case 1:  Bivariate distribution with continuous marginals,
+# - Case 2:  Bivariate distribution with continuous and discrete marginals,
+# - Case 3:  Trivariate distribution with continuous and discrete marginals,
+# - Case 4:  Extreme values conditioning.
+#
+# We illustrate the fact that the range of the conditioned distribution is updated.
+
+# %%
+import openturns as ot
+import openturns.viewer as otv
+import openturns.experimental as otexp
+
+ot.ResourceMap.SetAsString("Contour-DefaultColorMapNorm", "rank")
+
+# %%
+# Case 1: Bivariate distribution with continuous marginals
+# --------------------------------------------------------
+#
+# Here, we consider the bivariate distribution of :math:`(X_0, X_1)` defined by:
+#
+# .. math::
+#
+#   X_0 & \sim \Gamma(5, 0.5)\\
```

Comment:
There is no universal notation for each distribution: I suggest to explain what means the Gamma.